### PR TITLE
Display code coverage by dir in shipit frontend

### PR DESCRIPTION
**Pull request by La0**
* Load code coverage stats from task artifacts
* Serve easily through shipit-uplift
* Display in shipit frontend

### CODE REVIEW

File: src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import fnmatch
+import json
+import os
+import re
+import requests
+
+
+MAX_LEVEL = 1
```
Comment (marco-c) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: My fault, this should be 3.

Comment (marco-c) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: (I wrote this file, so perhaps you can review this one @La0?)

File: src/shipit_frontend/src/App/CodeCoverage.elm [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+
+view : Model -> Html Msg
+view model =
+    div [ class "container-fluid" ]
+        [ case model.directories of
+            NotAsked ->
+                div [ class "alert alert-info" ] [ text "Will soon display code coverage..." ]
+
+            Loading ->
```
Comment (marco-c) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Can we merge `Loading` and `NotAsked`?

File: src/shipit_frontend/src/App/CodeCoverage.elm [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        style =
+            if directory.current < directory.previous then
+                "table-danger"
+            else if directory.current > directory.previous then
+                "table-success"
+            else
+                "table-info"
+    in
+        tr [ class style ]
+            [ td [] [ span [ class "btn btn-link", onClick (SetDirectory (Just path)) ] [ text path ] ]
```
Comment (marco-c) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Can we make the cursor a pointer here?

**Comment by marco-c** [comment]
Somebody should review the Elm code, as I'm completely unfamiliar with it.

File: src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py (hunk 1 above)
Comment (marco-c) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: I fixed it.

File: src/shipit_frontend/src/App/CodeCoverage.elm (hunk 3 above)
Comment (La0) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Yes, done in scss.

**Comment by garbas** [comment]
I will merge this manually, to not squash to commits. In future lets follow "one commit" == "one PR" policy.

**Pull request closed without merging**